packages/appstore/src/modules/traders-hub/index.tsx

```diff
@@ -111,11 +111,19 @@ const TradersHub = observer(() => {
         );
     };
 
-    const mt5_ordered_platform_sections = is_mt5_allowed ? (
-        <OrderedPlatformSections />
-    ) : (
-        <OrderedPlatformSections is_cfd_visible={false} is_options_and_multipliers_visible={true} />
-    );
+    const getOrderedPlatformSections = (isDesktop = false) => {
+        if (is_mt5_allowed) {
+            return isDesktop ? (
```

Comment:
Do a early return for isDesktop so the rest of the code doesnt get executed

Reply:
it will be same as if else statement
```
if (Desktop) return <OrderedPlatformSections />
if (isMobile) return <OrderedPlatformSections
                    is_cfd_visible={selected_platform_type === 'cfd'}
                    is_options_and_multipliers_visible={selected_platform_type === 'options'}
                />
```